MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -385,18 +387,17 @@ protected void setPlaybackHeadUpdateInterval() {
     }
     if (playItemHaveVideoTrack) {
       Player.VideoComponent videoComponent = player.get().getVideoComponent();
-      videoComponent.setVideoFrameMetadataListener(frameRenderedListener);
-    } else {
-      // Schedule timer to execute, this is for audio only content.
-      updatePlayheadPositionTimer = new Timer();
-      updatePlayheadPositionTimer.schedule(new TimerTask() {
-        @Override
-        public void run() {
-          playerHandler.obtainMessage(ExoPlayerHandler.UPDATE_PLAYER_CURRENT_POSITION)
-              .sendToTarget();
-        }
-      }, 0, 15);
-    }
+      videoComponent.addVideoListener(videoListener);
+    }
+    // Schedule timer to execute, this is for audio only content.
+    updatePlayheadPositionTimer = new Timer();
+    updatePlayheadPositionTimer.schedule(new TimerTask() {
+      @Override
+      public void run() {
+        playerHandler.obtainMessage(ExoPlayerHandler.UPDATE_PLAYER_CURRENT_POSITION)
+            .sendToTarget();
+      }
+    }, 0, 15);
```

Comment:
15 seems pretty low, do we need it to be this low? I guess it's the same as we always did before, but just wondering